### PR TITLE
Sample

### PR DESCRIPTION
**Pull request by haohancom**
This PR just provide a samples app base on React to the webrtc core, not includes function implementations. Will invoke functions in `device-management.ts` once they are merged.

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "samples",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```
Comment (arun3528) [inline samples/package.json:6]: i am not sure if you need a package.json here , we will need to merger the code together or make them as monorepo

File: samples/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (arun3528) [inline samples/package.json:1]: also we dont need package.lock file here we are using yarn

File: samples/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
```
Comment (arun3528) [inline samples/package.json:28]: all these need to be converted to configs are moved to the main package.json

Where did you refer the samples from ?

**Review by arun3528 — CHANGES_REQUESTED** [review]

File: samples/package.json (hunk 3 above)
Comment (haohancom) [reply samples/package.json:28]: I use [create-react-app](https://create-react-app.dev/) to build this sample folder from zero. You are right, Arun, merge this `package.json` with the main one is better. Let me try it.

**Pull request closed without merging**